kolibri/plugins/perseus_viewer/assets/dist/README.md

```diff
@@ -0,0 +1,3 @@
+The files in this directory are generated using the buildPerseus script inside this plugin.
+They should be regenerated using the same command in the unlikely event that Perseus ever needs
+to be updated.
```

Comment:
eot and woff2 files need to be tracked in git LFS:

```bash
git lfs track "*.eod"
git lfs track "*.woff2"
```

ref: https://github.com/learningequality/kolibri/blob/release-v0.15.x/.gitattributes

Reply:
Ah, I thought git LFS had picked them up already, but I guess not.

Reply:
To remind myself, I am going to update this to only include `.woff` files, exclusively, and update the CSS accordingly - we determined that `.woff` is maximally compatible for the platforms we support.